src/pages/Dashboard/hooks/useTotalBalance/useTotalBalance.ts

```diff
@@ -15,7 +15,7 @@ export const useTotalBalance = (accounts: ReturnType<typeof flattenTokenBalances
     Object.values(accounts).forEach(account => {
       if (!account) return
       const identifier = account.contract ?? account.chain
-      const accountMarketData = marketData[identifier]
+      const accountMarketData = marketData.marketData[identifier]
```

Comment:
```suggestion
      const accountMarketData = marketData?.marketData?.[identifier]
```
maybe?

Reply:
nah not this one, the inner object is always defined and the string lookup will return undefined which gets caught below